server/src/db/pg-query.ts

```diff
@@ -18,13 +21,13 @@ import { MPromise } from "../utils/metered";
 //
 // Note we use native
 const usingReplica =
-  process.env.DATABASE_URL !==
-  process.env[process.env.DATABASE_FOR_READS_NAME as string];
+config.get('database_url') !==
+  config.get(config.get('database_for_reads_name').toLowerCase());
```

Comment:
I wonder if `database_for_reads_name` should now point to the lower case version for clarity?